code/src/SureQuestion.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+
+export const SureQuestion = ({ sure, setSure}) => {
+    const areYouSure = ["Yep! Pretty sure.", "Dunno", "Maybe not"];
+
+    return (
+      <div>
+        <h2>Is it really your favourite?</h2>
+        {areYouSure.map(sure => (
+          <label tabIndex="0">
+            <input
+              type="radio"
+              value={sure}
+              onChange={(event) =>  setSure(event.target.value)}
+            />
+            {sure}
+          </label>
+        ))}
+        </div> 
+    );
+}
+
+// Something is wrong in this function but I can't figure it out 
```

Comment:
Maybe you can try something like this: the options of the radio button are outside if the form:

    const areYouSure = [
           "Yep! Pretty sure.", 
           "Dunno", 
           "Maybe not"
    ];

export const SureQuestion = ({ sure, setSure}) => {
    return (
      <form>
        <h2>Is it really your favourite?</h2>
        {areYouSure.map(sure => (
          <label tabIndex="0" key={group}>
            <input
              type="radio"
              value={group}
              onChange={event =>  setSure(event.target.value)}
            />
            {group}
          </label>
        ))}
        </form> 
    );
}